src/modules/task/components/trigger/TelegramTriggerConfig.tsx

```
@@ -0,0 +1,453 @@
+            </a>
+          </div>
+        </div>
+      </div>
+
+      {/* Bot Token Input */}
+      <div>
+        <label className="block text-sm font-medium text-zinc-300 mb-2">
+          Bot Token <span className="text-red-400">*</span>
+        </label>
+        <p className="text-xs text-zinc-500 mb-3">
+          Get your bot token from <a href="https://t.me/BotFather" target="_blank" className="text-blue-400 hover:underline">@BotFather</a> by sending /newbot
```

Biome (2.3.14)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)